### DISCUSSION TITLE
Would you take a commission?

### DISCUSSION THREAD

**Discussion by garetmg** [Dakkaron/Fairberry#26] [General]
Hi, would you ever be willing to build one of these for me, and if so how much would you charge? I'd want one that works with S24 Ultra (bonus points if it didn't block the sPen and was black)

**Comment by XantiaActiva** [comment]
@garetmg where are you located? You are searching for someone to supply the complete keyboard if I uderstand correctly? Please keep in mind, that it will always look a little bit like a DIY solution. I tink the best solution is, if someone with the same mobile phone as yours builds the complete package. As I do not have CAD skills, I cannot alter cases in order to suit to other cell phones. What I can do is to program the circuit board and connect it to the PCB (soldering work). If a case is supplied I can assemble and test the configuration. 

What to charge for this? Good question. In case of my build:

- PCB: 45,-- EUR incl. shipping for 2 pc. --> 1 pc. = 22,50 EUR
- USB-C OTG plug --> 0,10 EUR
- Keyboard: 11,68 EUR
- Just lower part of the case with mounting bracket for the USB plug: 20,00 EUR 
- Cables and programming devices: 20 EUR, can be used for further projects; for this case 10,00 EUR

This sums up to 64,28 EUR only for parts and so on. Actually without work. I think that the projekt took me approx. 10 hours of work with reading, soldering, programming, assembling, etc. Due to the knowledge I now have the next time it could be much less, but I think approx. 3 hours. After that the keyboard has to be shipped --> additional cost. 

If time is no problem the complete thing will get cheaper. But who wants to stock 4 PCBs just to get a good price if you will most likely have no customer for these 4 PCBs?

**Reply by Dakkaron** [reply]
The main reason why the Fairberry is open  source is because I did the math and figured out that it's not worth doing commissions.

The Clicks keyboard sells for €130, and that's including the cost benefits of mass production.

For this to make any kind of economic sense, I'd have to take €200+ per piece if I also have to adjust the 3D model to fit the customer's phone. That's pretty tricky without having the phone in question at hand.

Factor in DOA units, returns, units lost in shipping and customer support and things get expensive real quick.

I was thinking about selling the electronics only, but then again ordering them from JLCPCB is the easiest part and if I have to handle, program, check, package and ship them, that's likely going to cost a fair bit more than to just order two (minimum order quantity) from JLCPCB and have a spare.

The math is just not mathing, and if I do that at some scale, I can't afford doing that as a charity.

So my charity aspect is that I give away the design files for free instead.

The only way I could figure out how the Fairberry makes financial sense is if you DIY build it for yourself as a hobby thing and don't pay for the time.
